fw/Core/Hitcon/App/SnakeMenuApp.h

```diff
@@ -0,0 +1,34 @@
+#ifndef SNAKE_MENU_APP_H
+#define SNAKE_MENU_APP_H
+
+#include "MenuApp.h"
+#include <App/SnakeApp.h>
+#include <App/ShowNameApp.h>
+#include <Logic/BadgeController.h>
+
+namespace hitcon {
+namespace app {
+namespace snake{
+
+constexpr menu_entry_t snake_menu_entries[] = {
+    {"Single Player", &snake_app, &SetSingleplayer},
```

Comment:
I'm thinking from a UX perspective we can consider triggering multiplayer mode from the menu that shows up when players connect.